target/linux/ramips/image/mt76x8.mk

```diff
@@ -257,6 +257,7 @@ define Device/tplink-safeloader
 	append-metadata | check-size $$$$(IMAGE_SIZE)
   IMAGE/factory.bin := append-rootfs | tplink-safeloader factory
 endef
+DEVICE_VARS += TPLINK_BOARD_ID
```

Comment:
Need to also add TPLINK_HEADER_VERSION

Reply:
Hm, why? For future use? 
TBH it makes sense, I had to debug quite some time why my TPLINK_BOARD_ID had no effect.

Reply:
I've just started to prepare a patch to unify tplink device definitions in a common file, this should be dealt with by that, so ignore it for now.